spec/unit/hanami/mailer/dsl_spec.rb

```diff
@@ -21,7 +21,7 @@
     describe 'when no value is set' do
       it 'returns a set of templates' do
         template_formats = LazyMailer.templates.keys
-        expect(template_formats).to eq(%i[html txt])
+        expect(template_formats).to match_array(%i[html txt])
```

Comment:
This test was failing for me. I got a different order

Reply:
Yeah, when we don't have a strict expectation on the order of an array, it's better to use `match_array` instead of `eq`. That will make specs less brittle.